Sources/Extensions/CoreGraphics/CGSizeExtensions.swift

```diff
@@ -50,5 +50,138 @@ public extension CGSize {
 		return CGSize(width: aWidth, height: aHeight)
 	}
 
+}
+
+// MARK: - Operators
+public extension CGSize {
+
+    /// SwifterSwift: Add two CGSize
+    ///
+    ///     let sizeA = CGSize(width: 5, height: 10)
+    ///     let sizeB = CGSize(width: 3, height: 4)
+    ///     let result = sizeA + sizeB
+    ///     //result = CGSize(width: 8, height: 14)
+    ///
+    /// - Parameters:
+    ///   - lhs: CGSize to add to.
+    ///   - rhs: CGSize to add.
+    /// - Returns: The result comes from the addition of the two given CGSize struct.
+    public static func + (lhs: CGSize, rhs: CGSize) -> CGSize {
+        return CGSize(width: lhs.width + rhs.width, height: lhs.height + rhs.height)
+    }
+
+    /// SwifterSwift: Add a CGSize to self.
+    ///
+    ///     let sizeA = CGSize(width: 5, height: 10)
+    ///     let sizeB = CGSize(width: 3, height: 4)
+    ///     sizeA += sizeB
+    ///     //sizeA = CGPoint(width: 8, height: 14)
+    ///
+    /// - Parameters:
+    ///   - lhs: self
+    ///   - rhs: CGSize to add.
+    public static func += (lhs: inout CGSize, rhs: CGSize) {
+        lhs = CGSize(width: lhs.width + rhs.width, height: lhs.height + rhs.height)
```

Comment:
I believe you can do in-place mutation here like:
```swift
       lhs.height += rhs.height
       lhs.width += rhs.width
```
Instead of creating a new instance of cgsize and make the attribution.

Reply:
Modified all of them.